sql/README.md

```diff
@@ -0,0 +1,141 @@
+GasBuddy SQL Standards
+==========
+
+
+naming
+========
+SQL views, indices, and stored procedures all being with a prefix which indicates their type. 
+Index names contain the name of the table followed by the columns in the index as well as other important information (e.g. "filtered", "covering", etc.). 
+Stored procedure names contain the table name, followed by the operation, followed by the version number.
+
+```
+views: 		[dbo].[vw_station_with_prices]
+indexes: 	[dbo].[idx_members_id]
+stored procedure which an individual object as output: 	[dbo].[usp_members_get_by_id]
+stored procedure which a list of objects as output: 	[dbo].[usp_member_comments_list_by_member_id]
+versioning:	[dbo].[usp_member_comments_list_by_member_id_v2]
+```
+
+SQL tables and view are plural, lower case, underscore separated. 
+
+```
+[dbo].[awards]
+[dbo].[members]
+[dbo].[member_comments]
+```
+
+
+formatting
+=========
+All T-SQL reserved words are uppercase.
+All identifier should be delimited by square brackets.
+
+```
+CREATE TABLE [dbo].[price_hist2](
+	[id] [BIGINT] IDENTITY(1,1) NOT NULL,
+	[ph2_event_id] [BIGINT] NOT NULL,
+	[ph_id] [BIGINT] NOT NULL,
+	[sm_id] [INT] NOT NULL,
+	[fuel_product_id] [SMALLINT] NOT NULL,
+	[price_type_id] [TINYINT] NOT NULL,
+	[price] [DECIMAL](5, 2) NOT NULL,
+	[source] [SMALLINT] NOT NULL,
+	[station_tme] [DATETIME] NOT NULL,
+	[server_tme] [DATETIME] NOT NULL,
+	[member_id] [VARCHAR](30) NOT NULL,
+	[bad_price] [BIT] NOT NULL
+);
+
+```
+
+For stored procedures: Add description to the script.
+All SQL statements should end with semicolon(;).
+All the references to identifiers should contain the schema name.
+Do not add database for local objects.
+Respect CASE used in identifiers and columns definition.
+
+```
+-- =============================================
+-- Author:		[Name]
+-- Create date: [mm/dd/yyyy]
+-- Description:	[add description]
+-- Update Log:
+--	VERSION		DATE			AUTHOR		DESCRIPTION
+--	 V#			 yyyy-mm-dd		 <name>		 <What changed?>
+-- =============================================
+CREATE PROCEDURE [dbo].[usp_fuel_products_get_by_id] 
+	@id INT
+AS
+BEGIN
+	SET NOCOUNT ON;
+
+	SELECT id
+		  ,display_name
+		  ,fuel_group_id
+		  ,octane_id
+		  ,ethanol_id
+		  ,category_id
+	FROM [dbo].[fuel_products] WITH(NOLOCK)
+	WHERE id = @id;
+
+END
+
+```
+
+When joining, use aliases for the tables and identify all the columns with their corresponding alias.
+
+```
+	SELECT		srdt.sm_id , 
```

Comment:
Is this using tabs over spaces? Might be worth saying all indenting to be done with spaces?